crates/pop-cli/src/commands/call/parachain.rs

```diff
@@ -628,6 +672,7 @@ mod tests {
 		assert_eq!(call_parachain.extrinsic.name, "place_order_allow_death");
 		assert_eq!(call_parachain.args, ["10000".to_string(), "2000".to_string()].to_vec());
 		assert_eq!(call_parachain.suri, "//Bob");
+		assert!(!call_config.sudo);
```

Comment:
```suggestion
		assert!(!call_parachain.sudo);
```

Reply:
Good catch, solved in https://github.com/r0gue-io/pop-cli/pull/349/commits/924662d302ac63a1e6d64a36d3849cf90fa06250